### PR TITLE
Updated Resources webpage with latest talks and links

### PR DESCRIPTION
**Pull request by agoyal26**
## Why are these changes needed?

Updated with latest links and resources for DPK

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -7,24 +12,43 @@
...
+7. **Talk and Hands on session** at [MIT Bangalore](https://www.linkedin.com/posts/saptha-surendran-71a4a0ab_ibmresearch-dataprepkit-llms-activity-7261987741087801346-h0no?utm_source=share&utm_medium=member_desktop)
+8. **PyData NYC 2024** - [90 mins Tutorial](https://nyc2024.pydata.org/cfp/talk/AWLTZP/)
+9. **Open Source AI** [Demo Night](https://lu.ma/oss-ai?tk=A8BgIt)
+10. [**Data Exchange Podcast with Ben Lorica**](https://thedataexchange.media/ibm-data-prep-kit/)
+11. Unstructured Data Meetup - SF, NYC, Silicon Valley
+12. IBM TechXchange Las Vegas
+13. Open Source [**RAG Pipeline workshop**](https://www.linkedin.com/posts/sujeemaniyam_dataprepkit-workshop-llm-activity-7256176802383986688-2UKc?utm_source=share&utm_medium=member_desktop) with Data Prep Kit at TechEquity's AI Summit in Silicon Valley
+14. **Data Science Dojo Meetup** - [video](https://datasciencedojo.com/tutorial/data-preparation-toolkit/)
+15. [**DPK tutorial and hands on session at IIIT Delhi**](https://www.linkedin.com/posts/cai-iiitd-97a6a4232_datascience-datapipelines-machinel[%E2%80%A6]65125349376-FG8E/?utm_source=share&utm_medium=member_desktop)
```
Comment (shahrokhDaijavad) [inline resources.md]: The link to 15 is broken. Please correct.

File: resources.md [hunk 2]
```diff
@@ -7,24 +12,43 @@
...
+- [**DPK Header Cleanser Module Blog by external contributor**](https://www.linkedin.com/pulse/enhancing-data-quality-developing-header-cleansing-tool-kalathiya-i1ohc/?trackingId=6iAeBkBBRrOLijg3LTzIGA%3D%3D) 
 
-## Workshops
 
-- **2024-09-21: "RAG with Data Prep Kit" Workshop** @ Mountain View, CA, USA - [info](https://github.com/sujee/data-prep-kit-examples/blob/main/events/2024-09-21__RAG-workshop-data-riders.md)
-
-## Discord
+# Relevant online communities
 
 - [**Data Prep Kit Discord Channel**](https://discord.com/channels/1276554812359442504/1286046139921207476)
```
Comment (shahrokhDaijavad) [inline resources.md]: The link to the discord channel should be changed to the "new" data-prep-kit channel. Please change to: 
https://discord.com/channels/1276554812359442504/1303454647427661866

Comment (agoyal26) [reply resources.md]: done

File: resources.md (hunk 1 above)
Comment (agoyal26) [reply resources.md]: It was active a few days ago- was posted by the institute themselves. They seemed to have taken it down. For now I have removed it - will see if I find an alternate.

**Review by shahrokhDaijavad — APPROVED** [review]
LGTM. Thanks, @agoyal26.

**Pull request merged**